src/pybind/mgr/balancer/module.py

```diff
@@ -327,6 +327,9 @@ class Module(MgrModule):
     no_optimization_needed = False
     success_string = 'Optimization plan created successfully'
     in_progress_string = 'in progress'
+    last_pg_upmap = []
+    added_pg_upmap_items = []
+    removed_pg_upmap_items = []
```

Comment:
The "make check" test is failing due to:
```
balancer/module.py: note: In class "Module":
balancer/module.py:330: error: Need type annotation for "last_pg_upmap" (hint: "last_pg_upmap: List[<type>] = ...")
balancer/module.py:331: error: Need type annotation for "added_pg_upmap_items" (hint: "added_pg_upmap_items: List[<type>] = ...")
balancer/module.py:332: error: Need type annotation for "removed_pg_upmap_items" (hint: "removed_pg_upmap_items: List[<type>] = ...")
```

It's saying to add typing to your code, since that is enforced here. This should fix it:
```suggestion
    last_pg_upmap: List[Dict[str, Any]] = []
    added_pg_upmap_items: List[Dict[str, Any]] = []
    removed_pg_upmap_items: List[Dict[str, Any]] = []
```